packages/react-recommendations/src/__tests__/RelatedProducts.tsx

```diff
@@ -0,0 +1,136 @@
+import { render, waitFor } from '@testing-library/react';
+import React from 'react';
+
+import { createRecommendationsClient } from '../../../../test/utils';
+import { RelatedProducts } from '../RelatedProducts';
+
+const { searchClient } = createRecommendationsClient();
+
+const ItemComponent = ({ item }) => <div>{item.name}</div>;
+
+describe('RelatedProducts', () => {
+  beforeEach(() => {
+    document.body.innerHTML = '';
+  });
+
+  test('render with default props', async () => {
+    render(
+      <RelatedProducts
+        // @ts-expect-error
+        searchClient={searchClient}
+        indexName={'indexName'}
```

Comment:
```suggestion
        indexName="indexName"
```

Reply:
Fixed in https://github.com/algolia/ui-components/pull/26/commits/ece3ce07a79935145c7bfef4488f6525aaa4355a